### PR TITLE
Feature/create

### PR DESCRIPTION
**Pull request by kaniak274**
This pr context is to add mutations to:
-  Create book
-  Create author

plus it add's localstack to the development environment, so we can mock AWS environment for free and send files (photo of book or author) to the fake s3 container.

This pr does not:
1. Validate data i.e if passed author id points to the author.
2. Does not have unit tests. I will add them when I write some validators.

### CODE REVIEW

File: docker/development/docker-compose.yml [hunk 1]
```diff
@@ -27,14 +27,35 @@ services:
             - books
         environment:
             DATABASE_URI: postgresql://books:secret@books-db:5432/books
+            AWS_BOOKS_BUCKET_NAME: books
+            AWS_AUTHORS_BUCKET_NAME: authors
+            AWS_ENDPOINT_URL: http://localstack:4566
```
Comment (KamilSupera) [inline docker/development/docker-compose.yml:32]: For the future can we add variables for this kind of data. I mean to add a variable for aws_endpoit_url  or db uri

File: requirements/base.txt [hunk 2]
```diff
@@ -7,3 +7,5 @@ Flask-GraphQL==2.0.1
 
 Inject==4.1.1
 graphene==2.1.8
+
```
Comment (KamilSupera) [inline requirements/base.txt:10]: This line is needed or add commet what is this or section

File: docker/development/docker-compose.yml (hunk 1 above)
Comment (kaniak274) [reply docker/development/docker-compose.yml:32]: Variables? What do you mean. Could you explain more

File: requirements/base.txt (hunk 2 above)
Comment (kaniak274) [reply requirements/base.txt:10]: I just enter i doesn't do anything :)

Comment (kaniak274) [reply requirements/base.txt:10]: It just enter it doesn't mean anything

**Review by KamilSupera — APPROVED** [review]

**Pull request merged**